VideoLocker/src/androidTest/java/org/edx/mobile/test/feature/config/E2EConfig.java

```diff
@@ -0,0 +1,6 @@
+package org.edx.mobile.test.feature.config;
+
+public class E2EConfig {
+    public static String EMAIL_TEMPLATE = "test-{unique_id}@example.com";
```

Comment:
We need to add values to the ``.yaml`` files for this (using the ones from the wiki), but the values themselves should be read from the config. So ``EMAIL_TEMPLATE`` should be an instance variable ``emailTemplate``, and ``Config`` should have an ``endToEndConfig`` member. Is that clearer